### PR TITLE
[Doc]Update mmdet.md(cn/en)

### PR DESCRIPTION
**Pull request by Luo-Z13**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by irexyc** [comment]
Hi @Luo-Z13 , Please fix the lint error, you can refer to [this](https://pre-commit.com/) for how to use it.

**Comment by Luo-Z13** [comment]
> Hi @Luo-Z13 , Please fix the lint error, you can refer to [this](https://pre-commit.com/) for how to use it.

Thanks, I'm trying...

**Comment by Luo-Z13** [comment]
> Hi @Luo-Z13 , Please fix the lint error, you can refer to [this](https://pre-commit.com/) for how to use it.

Hello, It seems I fix the line error, but "files changed" seems different... I just change 4 lines of mmdet.md in en and zh_cn, but it shows some redundancy.

File: tests/test_codebase/test_mmdet3d/data/kitti/training/velodyne_reduced/000008.bin
[the remark this answers is not in the document]
Comment (Luo-Z13) [reply tests/test_codebase/test_mmdet3d/data/kitti/training/velodyne_reduced/000008.bin:1]: Sry, I didn't modify it locally on purpose, I'll correct it.

[the remark this answers is not in the document]
Comment (RunningLeon) [reply tests/test_codebase/test_mmdet3d/data/kitti/training/velodyne_reduced/000008.bin:1]: @Luo-Z13 hi, can you save the changed markdown files to another place and revert all commits and then add the new commit to only change the markdown files?

**Comment by RunningLeon** [comment]
@Luo-Z13 hi, could kindly resolve conflicts?

**Comment by Luo-Z13** [comment]
> @Luo-Z13 hi, could kindly resolve conflicts?

I resolve conflicts, and in https://github.com/open-mmlab/mmdeploy/pull/2054 .

**Pull request closed without merging**